plugins/tiles/src/main/java/org/apache/struts2/tiles/StrutsTilesContainerFactory.java

```diff
@@ -105,19 +102,8 @@ public class StrutsTilesContainerFactory extends BasicTilesContainerFactory {
 
     /**
      * Default pattern to be used to collect Tiles definitions if user didn't configure any
-     *
-     * @deprecated since Struts 6.4.0, use {@link #TILES_DEFAULT_PATTERNS} instead
      */
-    @Deprecated
-    public static final String TILES_DEFAULT_PATTERN = "/WEB-INF/**/tiles*.xml,classpath*:META-INF/**/tiles*.xml";
-
-    /**
-     * Default pattern to be used to collect Tiles definitions if user didn't configure any
-     */
-    public static final Set<String> TILES_DEFAULT_PATTERNS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
```

Comment:
Technically you need to deprecate this since it's been released in 6.4.0. But also there's no need to create the set anew each time below. You can just keep this field as is and update the value like so:

```public static final Set<String> TILES_DEFAULT_PATTERNS = TextParseUtil.commaDelimitedStringToSet("*tiles*.xml");```

Reply:
Right, fixed